msprime/ancestry.py

```diff
@@ -1861,7 +1861,7 @@ class SmcKApproxCoalescent(ParametricAncestryModel):
     hull_offset: float
 
     # We have to define an __init__ to enforce keyword-only behaviour
-    def __init__(self, *, duration=None, hull_offset=0.0):
+    def __init__(self, *, duration=None, hull_offset):
```

Comment:
I think we should make hull offset a required positional arg, like
```
def __init__(self, hull_offset, *, duration=None):
    ...
```

I don't think we put much thought into this when it was going in initially, just put something in to get things working.

(unless this breaks other things?)